src/ResponseEmitter.php

```diff
@@ -0,0 +1,121 @@
+<?php
+namespace GuzzleHttp\Psr7;
+
+use Psr\Http\Message\ResponseInterface;
+
+/**
+ * Response emitter
+ */
+class ResponseEmitter
```

Comment:
how about creating an Interface for this?

Reply:
Maybe. Let's decide all together?